uint32 id
float64 R
float64[] pose0
front_following/Path[] paths
